porting/index.md

```diff
@@ -179,9 +179,9 @@ After you built `hybris-boot.img` it is time to test it out.
  2. Start the device in bootloader (aka "fastboot") mode.<br>
  This is usually acomplished by holding some combination of keys on the device for a few seconds. On some devices this is Volume-Down + Power. The Lineage OS wiki is a good resource to check for this.
  3. Change to the directory where the boot images have been created with the command `cout`.
- 4. Boot the kernel image with: `fastboot boot hybris-boot.img`
-
-**//TODO: what are we supposed to do with the systemimage? What can we expect from it?**
+ 4. Flash the kernel image with: `fastboot flash boot hybris-boot.img`
+ 5. Reboot into the recovery, you can use the TWRP recovery latest version to deploy the rootfs and system.img, you can read the [following guide](deploying-rootfs.md)
```

Comment:
Next reboot into the recovery. You can use the latest version of TWRP recovery to deploy the rootfs and system.img. If you have further questions, check the [following guide](deploying-rootfs.md).